.github/workflows/generate-summary.yaml

```
@@ -40,8 +40,29 @@ on:
         description: 'weekly run (Optional)'
 
 jobs:
+  check-issue-name:
+    if: always()
+    runs-on: ubuntu-20.04
+    environment: production
+    defaults:
+      run:
+        working-directory: riscv-gnu-toolchain
+    steps:
+      - name: Check name
+        id: stop-issue-generation
+        run: |
+          if [ '${{ inputs.prefix }}' != '' ] && [[ '${{ inputs.issue_hash_prefix }}' =~ ^Testsuite\ Status\ [0-9a-f]{40}$ ]];
```

I forgot that issue_hash_prefix does not contain the hash.
```suggestion
          if [ '${{ inputs.prefix }}' != '' ] && [[ '${{ inputs.issue_hash_prefix }}' = "Testsuite Status" ]];
```